src/app-layout/visual-refresh-toolbar/index.tsx

```diff
@@ -463,7 +464,11 @@ const AppLayoutVisualRefreshToolbar = React.forwardRef<AppLayoutProps.Ref, AppLa
     return (
       <>
         {/* Rendering a hidden copy of breadcrumbs to trigger their deduplication */}
-        {!hasToolbar && breadcrumbs ? <ScreenreaderOnly>{breadcrumbs}</ScreenreaderOnly> : null}
+        {!hasToolbar && breadcrumbs ? (
+          <AppLayoutVisibilityContext.Provider value={isIntersecting}>
```

Comment:
How about wrapping the entire content into `AppLayoutVisibilityContext.Provider`, in case if we want to read somewhere else in the future without requiring a merge from live?